cli/flox/src/utils/search.rs

```diff
@@ -233,12 +253,21 @@ impl Display for DisplaySearchResults {
 
         while let Some(d) = items.next() {
             let desc = d.description.as_deref().unwrap_or(DEFAULT_DESCRIPTION);
-            write!(f, "{d:<column_width$}  {desc}", d = d.to_string())?;
+
+            let name = if self.bold_search_term {
```

Comment:
Suggestions:
- In the original ticket I suggested using the `console` crate to provide easier styling, I'd rather us not have to manage escape codes directly. You can see in the `console` README how to apply styling, it's pretty straightforward.
- I think you can use `std.replace` to replace `search_term` with a bolded version of the search term rather than splitting and looping over the string. This is simpler and less error prone.

If you do a `flox search hello` you'll see that the alignment of the columns no longer matches up. I haven't looked into it, but my intuition points at the splitting/looping as the culprit.

Reply:
> In the original ticket I suggested using the console crate to provide easier styling, I'd rather us not have to manage escape codes directly. You can see in the console README how to apply styling, it's pretty straightforward.

Ack.

> I think you can use std.replace to replace search_term with a bolded version of the search term rather than splitting and looping over the string. This is simpler and less error prone.

Good catch!

> If you do a flox search hello you'll see that the alignment of the columns no longer matches up. I haven't looked into it, but my intuition points at the splitting/looping as the culprit.

The alignment of the columns no longer matches up because the `column_width` is calculated based on the original string lengths without considering the additional characters introduced by the bolding.

When you replace the search term with a bolded version, the length of the string increases due to the additional formatting characters. However, the `column_width` is still calculated based on the original string lengths, which doesn't account for the extra characters.

I should have noticed this earlier, apologies.